### PR TITLE
Intern Blog Post - Maria Maryam

### PR DESCRIPTION
**Pull request by MariaM0003**
I wrote a blog post about the internship experience. Made a markdown and HTML file. Added my post to blog.html.

### CODE REVIEW

**Review by WrathfulSpatula — DISMISSED** [review]
LGTM!

File: posts/_markdown/intern_maria_maryam_post.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+date: 09 08 2022
+author: Maria Maryam
+---
+
+I was introduced to the Unitary Fund through [The Coding School’s](https://the-cs.org/)- [Qubit x Qubit](https://www.qubitbyqubit.org/) Early Quantum Career Immersion Program for undergraduate students. In June 2022, After three weeks of rigorous quantum computing training with Qubit x Qubit, I began my seven-week internship with the Unitary Fund.  When I applied to the program and began learning about quantum computing, I had very little knowledge and experience in both quantum and classical computing. After learning the basics of quantum computing, I was eager to start my work with Unitary Fund and learn even more through hands-on work. 
+
+
+## MY BACKGROUND
+
+I am a second-year student at CUNY Queens College where I am double majoring in Computer Science and Linguistics.  I am interested in machine learning, NLP, quantum computing, and any form of software development that is able to reach and help people
```
Comment (nathanshammah) [inline posts/_markdown/intern_maria_maryam_post.md:12]:
```suggestion
I am a second-year student at CUNY Queens College where I am double majoring in Computer Science and Linguistics.  I am interested in machine learning, NLP, quantum computing, and any form of software development that is able to reach and help people.
```

File: posts/_markdown/intern_maria_maryam_post.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## THE INTERNSHIP
+
+The first two weeks of my internship were part-time where I got to know the team and the projects better. Our first assignment was to make ten submissions to [Metriq](https://metriq.info/), in a week. This required a ton of reading through research papers, trying to understand tensor networks, and attempting to extract results from jargon-filled and oversaturated papers. Although it was not easy, it was incredibly beneficial for multiple reasons. The obvious reason is that I learned a great deal about quantum computing and scientific papers in general. I also got very familiar with the user interface of [Metriq](https://metriq.info/), which would be very beneficial for me in the coming weeks.
```
Comment (nathanshammah) [inline posts/_markdown/intern_maria_maryam_post.md:16]:
```suggestion
The first two weeks of my internship were part-time, during which I got to know the team and the projects better. Our first assignment was to make ten submissions to [Metriq](https://metriq.info/), in a week. This required a ton of reading through research papers, trying to understand tensor networks, and attempting to extract results from jargon-filled and oversaturated papers. Although it was not easy, it was incredibly beneficial for multiple reasons. The obvious reason is that I learned a great deal about quantum computing and scientific papers in general. I also got very familiar with the user interface of [Metriq](https://metriq.info/), which would be very beneficial for me in the coming weeks. 
```

**Review by nathanshammah — DISMISSED** [review]
It's a vivid and engaging account. Truly inspiring! ✨

**Review by nathanshammah — DISMISSED** [review]

File: posts/_markdown/intern_maria_maryam_post.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+title: Meet the UF Interns – Maria Maryam
+date: 09 08 2022
```
Comment (nathanshammah) [inline posts/_markdown/intern_maria_maryam_post.md:3]:
```suggestion
date: 22 08 2022
```

File: posts/intern_maria_maryam_post.html [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+                            </button>
+                        </div>
+                        <section class="light">
+
+
+                            <right>
+                                <h2 class="leading-arrow">Meet the UF Interns – Maria Maryam</h2>
+                                <h3></h3>
+                            </right>
+                            2022-08-09
```
Comment (nathanshammah) [inline posts/intern_maria_maryam_post.html:75]:
```suggestion
                            2022-08-22
```

**Review by nathanshammah — APPROVED** [review]

**Pull request merged**